frontend/ui/widget/dictquicklookup.lua

```diff
@@ -807,6 +808,7 @@ function DictQuickLookup:_instantiateScrollWidget()
     if self.is_html then
         self.shw_widget = ScrollHtmlWidget:new{
             html_body = self.definition,
+            html_resource_directory = util.splitFilePathName(self.dictionary_file),
```

Comment:
If `self.dictionary_file` is `nil`, `splitFilePathName` will return `"", ""`, which could be problematic later:
```suggestion
            html_resource_directory = self.dictionary_file and util.splitFilePathName(self.dictionary_file),
```